src/palace/manager/feed/serializer/opds.py

```diff
@@ -405,13 +413,95 @@ def _serialize_data_entry(self, entry: DataEntry) -> etree._Element:
     def to_string(cls, element: etree._Element) -> str:
         return cast(str, etree.tostring(element, encoding="unicode"))
 
+    @abc.abstractmethod
     def content_type(self) -> str:
```

Comment:
Minor: Since subclasses are expected to implement these, its would be nice to have a doc comment with a small bit of explanation of what an implementation should provide for this function.